tests/address.rs

```diff
@@ -2,6 +2,8 @@ use ethereum_types::H160;
 use hex_literal::hex;
 use rstest::rstest;
 
+// are we testing anything in rusty-rlp here?
```

Comment:
No, all the rust tests are just on the underlying rlp library to help me better understand it and validate my assumptions. The tests to `rusty-rlp` are all in `python_tests.py` and you can run them with `make test`